soccer/gameplay/generate_fsm_diagrams.py

```diff
@@ -21,8 +20,20 @@ def mkdir_p(path):
 logging.getLogger().setLevel(logging.INFO)
 
 import main
+import ui.main
 main.init()
 
+class moc_Point():
```

Comment:
Yep, that's pretty much it. Most of the python code can run completely independently of soccer just by importing the robocup.so file as a python module, which contains every class, function, etc that we setup with boost python bindings. At some point, I should probably write up a little more documentation on how that stuff works since it's pretty tough to figure out just from the code.

One thing you can do if you want to play around with it is to run python from the robocup-software directory with the appropriate paths added. It can be useful for testing code out quickly or viewing documentation.
```shell
make # ensure robocup.so python module is compiled and put in ./run
PYTHONPATH=run:soccer/gameplay python
```

Then you can use run robocup code in the python console:
```python
import robocup
p = robocup.Point(0.5, 0.5)
help(p)
help(robocup.SystemState)

import behavior
help(behavior)
```